frontend/src/components/organisms/ManageWebsite.tsx

```diff
@@ -0,0 +1,89 @@
+import React from "react";
+import Button from "../atoms/Button";
+import { api } from "@/utils/api";
+import Card from "../molecules/Card";
+
+const ManageWebsite = () => {
+  function convertToCSV(data: any[]) {
+    const csvArray = [];
+  
+    // Add header row
+    const headers = Object.keys(data[0]);
+    csvArray.push(headers.join(","));
+  
+    // Add data rows
+    data.forEach(item => {
+      const values = headers.map(header => item[header]);
+      csvArray.push(values.join(","));
+    });
+  
+    // Join rows with newline character
+    return csvArray.join("\n");
+  }
+
+  function createCSVFromArray(csvDataArray: any[]) {
+    // Combine all CSV data into one string
+    const combinedCSVData = csvDataArray.join('\n');
+  
+    // Create a Blob object from the combined CSV data
+    const blob = new Blob([combinedCSVData], { type: 'text/csv' });
+  
+    // Create a temporary anchor element to trigger the download
+    const anchor = document.createElement('a');
+    anchor.href = URL.createObjectURL(blob);
+    anchor.download = 'database.csv';
+    anchor.click();
+  
+    // Clean up
+    URL.revokeObjectURL(anchor.href);
+  }
+
+  const handleDownloadDatabase = async () => {
```

Comment:
Please when all is refactored - remember to wrap this in useQuery hook, that way we can better manage, loading, data, success and error states